src/Sylius/Bundle/AdminBundle/Resources/views/ProductVariant/Tab/_details.html.twig

```diff
@@ -2,8 +2,18 @@
     <h3 class="ui dividing header">{{ 'sylius.ui.details'|trans }}</h3>
 
     <div class="ui segments">
-        <div class="ui segment">
-            {{ form_row(form.name) }}
+        <div class="ui styled fluid accordion">
+            {% for locale, translationForm in form.translations %}
+                <div data-locale="{{ locale }}">
+                    <div class="title{% if 0 == loop.index0 %} active{% endif %}">
+                        <i class="dropdown icon"></i>
+                        <i class="{{ locale|slice(3, 2)|lower }} flag"></i> {{ locale|sylius_locale_name }}
+                    </div>
+                    <div class="ui content{% if 0 == loop.index0 %} active{% endif %}">
+                        {{ form_row(translationForm.name) }}
+                    </div>
+                </div>
+            {% endfor %}
```

Comment:
I noticed that almost all translation forms rendering look the same, maybe we could extract it to a macro?